.buildkite/pipelines/pipeline.kibana-release.yaml

```diff
@@ -0,0 +1,10 @@
+steps:
+  - label: ":releasethekraken: Release kibana"
+    # https://regex101.com/r/ScgA8Y/1
+    if: 'BUILDKITE_TAG =~ "/^v(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$$/
```

Comment:
kibana deploys will follow deploy@1692763619 pattern.  Ill capture this in a regex with some potential +/- cases.

Do we want a tag like this to flow qa -> staging -> prod @watson?

Reply:
Yes that's correct. When a tag like that is set on a commit in the Kibana repo, it should kick off the process of promoting that commit to QA through Staging all the way to Production